cli_helpers/tabular_output/tabulate_adapter.py

```diff
@@ -3,16 +3,17 @@
 
 from cli_helpers.packages import tabulate
 from cli_helpers.utils import filter_dict_by_key
-from .preprocessors import (bytes_to_string, align_decimals,
-                            override_missing_value)
+from .preprocessors import (convert_to_string, align_decimals,
```

Comment:
So, `convert_to_string` uses `bytes_to_string` behind the scenes. But, in order for the styling to work, every field in the data has to be a string. So, instead of only converting bytes, we need to convert everything (integers, too).

This reminds me, `align_decimals` needs to be run *before* `convert_to_string`, otherwise it won't work right (since `align_decimals` checks each fields' type).

I'll fix that, too.